app/src/main/java/com/movies/common/extensions/FragmentExtension.kt

```diff
@@ -59,21 +36,41 @@ fun <T> Fragment.observeLiveData(
     }
 }
 
+fun BaseFragment<*, *>.tryConfigureBottomView(tryAction: () -> Unit) {
+    try {
+        tryAction.invoke()
+    } catch (_: Exception) {
+        (parentFragment as BaseChildFragment).addBottomContainerView(bottomView())
+    }
+}
+
 fun <T> Fragment.changeScreen(fragment: Fragment, args: T? = null) {
     args?.let {
         val bundle = Bundle().apply {
-            putArguments(args)
+            putArguments(MOVIE_ID, args)
         }
         fragment.arguments = bundle
     }
 
     parentFragmentManager.beginTransaction().add(
-        R.id.nav_host_fragment, fragment, fragment.tag
-    ).addToBackStack(fragment.tag).commit()
+        R.id.childFragmentContainerView, fragment
+    ).addToBackStack(null).commit()
```

Comment:
why null ? what will happen if you write null instead of fragment.tag ?

Reply:
If you write a tag then you should specify tag in the xml. otherwise, it will have same effect in the backstack as null.